frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -37,9 +40,18 @@ public class Model {
     private int maxWorkers;
     private int batchSize;
     private int maxBatchDelay;
+    private int parallelLevel = 1;
```

Comment:
We need deprecate model json config gradually.  This release is in transition phase. the priority order is
TorchServe now uses a priority order to determine the final value of a model's parameters in frontend. Specifically, the config.property file has the lowest priority, followed by the model configuration YAML file, and finally, the REST or gRPC model management API has the highest priority.

Reply:
You mean the  json configs in the mar file or which json configs? I am referring to the ModelConfig that this pr introduces. Instead of unpacking the single values we should just give the whole structure as a parameter as we’re using most of the parameter anyways.